README.md

```diff
@@ -23,23 +23,34 @@ cargo install cargo-wipe
 cargo wipe --help
 ```
 
-## target
+## Usage
+
+To find build folders for `language` that can potentially be deleted run
 
 ```
-cargo wipe target
+cargo wipe language
 ```
 
-- Use the `-i <path>` argument to ignore certain paths.
-- Add the `-w` flag to wipe all Rust `target` folders found. USE WITH CAUTION!
+where `language` is `rust` or `node`. For example:

```

Comment:
Where referring to `language`, does it make sense to type as `<language>` just so that it's obvious that it's a parameter, not an actual value?